content/en/docs/tasks/debug-application-cluster/local-debugging.md

```diff
@@ -4,58 +4,56 @@ content_type: task
 
 
 <!-- steps -->
 
-## Getting a shell on a remote cluster
-
-Open a terminal and run `telepresence` with no arguments to get a `telepresence` shell. This shell runs locally, giving you full access to your local filesystem.
-
-The `telepresence` shell can be used in a variety of ways. For example, write a shell script on your laptop, and run it directly from the shell in real time. You can do this on a remote shell as well, but you might not be able to use your preferred code editor, and the script is deleted when the container is terminated.
+## Connecting your local machine to a remote Kubernetes cluster
+ 
+After installing `telepresence`, run `telepresence connect` to launch it's Daemon and connect your local workstation to the cluster.
 
-Enter `exit` to quit and close the shell.
+```
+$ telepresence connect
+ 
+Launching Telepresence Daemon
+...
+Connected to context default (https://<cluster public IP>)
+```
 
+You are now connected to the network of your remote cluster and so you can curl services using the Kubernetes syntax e.g. `curl -ik https://kubernetes.default`
+ 
 ## Developing or debugging an existing service
 
 When developing an application on Kubernetes, you typically program or debug a single service. The service might require access to other services for testing and debugging. One option is to use the continuous deployment pipeline, but even the fastest deployment pipeline introduces a delay in the program or debug cycle.
+ 
+Use the `telepresence intercept $SERVICE_NAME --port $LOCAL_PORT:REMOTE_PORT` command to create an "intercept" for rerouting remote service traffic.
+ 
+Where:
 
-Use the `--swap-deployment` option to swap an existing deployment with the Telepresence proxy. Swapping allows you to run a service locally and connect to the remote Kubernetes cluster. The services in the remote cluster can now access the locally running instance.
+-   `$SERVICE_NAME`  is the name of your local service
+-   `$LOCAL_PORT` is the port that your service is running on your local workstation
+-   And `$REMOTE_PORT` is the port your service listens to in the cluster
 
-To run telepresence with `--swap-deployment`, enter:
-
-`telepresence --swap-deployment $DEPLOYMENT_NAME`
-
-where $DEPLOYMENT_NAME is the name of your existing deployment.
-
-Running this command spawns a shell. In the shell, start your service. You can then make edits to the source code locally, save, and see the changes take effect immediately. You can also run your service in a debugger, or any other local development tool.
+Running this command will tell Telepresence to send remote traffic to your local service instead of the service in the remote Kubernetes cluster.  With this, you can then make edits to your service source code locally, save, and see the corresponding changes when accessing your remote application take effect immediately. You can also run your local service using a debugger or any other local development tool.
```

Comment:
```suggestion
Running this command tells Telepresence to send remote traffic to your local service instead of the service in the remote Kubernetes cluster. Make edits to your service source code locally, save, and see the corresponding changes when accessing your remote application take effect immediately. You can also run your local service using a debugger or any other local development tool.
```